modules/success_message.js

```diff
@@ -0,0 +1,13 @@
+// Show "✅ Add successfully" message after adding a book, hide it after 1.4s using setTimeout()
+// Add a class to the recent added book
+
+export default function showHideSuccessMessage(
```

Comment:
- [ ] It would be better to use arrow functions here too, as this is a better practice in ES6, please do this for all the functions.